lib/not-found.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const getTranslations = (translate) => {
```

Comment:
we don't need these `(` `)`'s. Not saying we should definitely remove them, guess its a preference thing but we should be consistent.